.travis.yml

```diff
@@ -1,24 +1,39 @@
 dist: xenial
 language: python
 cache: pip
-python:
-    - 2.7
-    - 3.7
 matrix:
-    allow_failures:
-        - python: 3.7
+  include:
+    - name: "Python 2.7 on Linux"
+      python: 2.7
+    - name: "Python 3.7 on Linux"
+      python: 3.7
+    - name: "Python 2.7 on macOS"
+      os: osx
+      osx_image: xcode10.2
+      language: shell  # 'language: python' is not yet supported on macOS
+      before_install: HOMEBREW_NO_AUTO_UPDATE=1 brew install npm
+    - name: "Python 2.7 on Windows"
+      os: windows
+      language: node_js
+      node_js: 12  # node
+      env: PATH=/c/Python27:/c/Python27/Scripts:$PATH
+      before_install: choco install python2
+  allow_failures:
+    - os: windows
+    - python: 3.7
```

Comment:
are we expecting failures on windows now that #1793 is landed?

Reply:
Python 3 yes but Windows no.  See comments below.